internal/command/testdata/show-json/nested-modules/output.json

```diff
@@ -68,7 +68,7 @@
                       "mode": "managed",
                       "type": "test_instance",
                       "name": "test",
-                      "provider_config_key": "more:test",
+                      "provider_config_key": "module.my_module.module.more:test",
```

Comment:
Here is an example of how the previous version of these opaque keys were incorrect. If two cousin modules were named "more", these values would clash, and either way there would never be an entry in the provider config map for `"more:test"`.